app/src/components/home/Home.js

```diff
@@ -38,18 +37,9 @@ class Home extends Component {
   };
 
   handleSubmit = () => {
-    //Only carryout submission if string is present
-    if(this.props.query.current !== ''){
-      if(this.props.query.current.length < 3) {
-          openSnackbar({ message: 'Minimum length of Search word must be 3 characters.' });
-          //TODO - clean this up, return easy to miss and is important for sidestepping action trigger
-          return
-      }
-      this.props.queryActions.submitQuery();
+      this.props.queryActions.submitQuery(this.props.query.current);
       //TODO:DM - should we also do a fetch here? we do in menu bar for cases where the history push doesn't change route handler
```

Comment:
can get rid of this TODO comment, maybe add a regular comment explaining that all we need to do in the component is to trigger the action... it will be determined whether it was a valid query from there